core/commands/checkout.py

```diff
@@ -37,7 +37,12 @@ def on_branch_selection(self, branch):
 
         self.git("checkout", branch)
         sublime.status_message("Checked out `{}` branch.".format(branch))
-        util.view.refresh_gitsavvy(self.window.active_view(), refresh_sidebar=True)
+        refresh_all_visible_views(self.window)
+
+def refresh_all_visible_views(window):
```

Comment:
Can you move this into [common/util/view.py](https://github.com/divmain/GitSavvy/blob/master/common/util/view.py#L81)

Reply:
Done.